README.md

```diff
@@ -16,16 +16,18 @@ Manual Update / Restore
 Restoring on Linux/OS X
 ------
 
-1. Open a terminal
-2. Clone this repo `git clone https://github.com/commaai/eon-neos.git`, then cd into the cloned repo `cd eon-neos`
-3. Run `./download.py`
-4. Put your Eon into fastboot mode by turning off your Eon, then holding volume down + power (comma two, eon gold), or volume up + power (eon).
-5. Run `./flash.sh` DO NOT DISCONNECT THE DEVICE!
+1. Connect your Comma Two/EON via a USB-C to USB-A cable into your computer (a.k.a host machine)
```

Comment:
That's correct for a Comma Two, but EONs have mini-B instead of C

Reply:
fixed